app/controllers/middleware_server_controller.rb

```diff
@@ -45,6 +45,12 @@ class MiddlewareServerController < ApplicationController
                                     :hawk  => N_('Not deploying to Hawkular server'),
                                     :msg   => N_('Deployment initiated for selected server(s)'),
                                     :param => :file
+    },
+    :middleware_add_jdbcdriver  => {:op    => :add_middleware_jdbc_driver,
```

Comment:
Right.. I didn't separated so that would not have to change all the others to align. But it's probably best to do it.